### ISSUE TITLE
evaluate report from OBO dashboard

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/cell-ontology#632]
http://obo-dashboard-test.ontodev.com/

Report here: http://obo-dashboard-test.ontodev.com/cl/dashboard.html

**Comment by addiehl** [comment]
I will create a separate issue to fix the duplicate definition items, several of which are simply duplicate terms, requiring obsoleting one in favor of the other.

Other aspects of the report, particularly the missing definitions, will take longer to resolve and may require multiple issues to be created.

**Comment by nicolevasilevsky** [comment]
Here is a [spreadsheet](https://docs.google.com/spreadsheets/d/1Bcu6O5hgqAxTMcha_pDv4yZCQ8kTlMiLlb8o4b_2He0/edit#gid=1096481151)

**Comment by nicolevasilevsky** [comment]
@addiehl I revised all the CL terms with duplicate defs, could you review/approve my PR?

**Comment by cmungall** [comment]
Where do we stand on this? We want to fix all these so we can instigate stricter travis checks

**Comment by nicolevasilevsky** [comment]
we've made some progress on it. Can you re-run the report so we can see what is left?

**Comment by nicolevasilevsky** [comment]
Looking at my [spreadsheet](https://docs.google.com/spreadsheets/d/1Bcu6O5hgqAxTMcha_pDv4yZCQ8kTlMiLlb8o4b_2He0/edit#gid=308594775), it looks like there are a lot of classes without text definitions.

Maybe the best approach would be to add text defs via patterns, where possible?

**Comment by nicolevasilevsky** [comment]
Oh, it actually looks like I started doing that, in this [tab](https://docs.google.com/spreadsheets/d/1Bcu6O5hgqAxTMcha_pDv4yZCQ8kTlMiLlb8o4b_2He0/edit#gid=308594775)

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by addiehl** [comment]
I think patterns will work for many of these, but not all.  Many of these go back to the original CL, and for some the intended meaning may be indiscernible.

**Comment by nicolevasilevsky** [comment]
Agreed- doing a quick look now, and some definitely conform to pattern, and there may be some that I can't figure out what they are, like CL:0000204 acceleration receptive cell. May be a candidate for obsoletion.

**Comment by addiehl** [comment]
There is this: https://www.jneurosci.org/content/24/35/7690
Visual Neurons in the Pigeon Brain Encode the Acceleration of Stimulus Motion

And this: https://snl.salk.edu/~fklam/pubs/Klam&Graf2003.pdf
Vestibular response kinematics in posterior parietal cortex neurons of macaque monkeys

but these papers describe two different types of neurons that may be involved in detection/perception of acceleration.

**Comment by dosumis** [comment]
> There is this: https://www.jneurosci.org/content/24/35/7690
> Visual Neurons in the Pigeon Brain Encode the Acceleration of Stimulus Motion
> 
> And this: https://snl.salk.edu/~fklam/pubs/Klam&Graf2003.pdf
> Vestibular response kinematics in posterior parietal cortex neurons of macaque monkeys
> 
> but these papers describe two different types of neurons that may be involved in detection/perception of acceleration.

Reasonable in have parent term defined functionally (with GO) and add 2 subtypes based on papers (with refs).

**Comment by nicolevasilevsky** [comment]
I'll add this to the CL meeting agenda. It is going to be a lot of work to add definitions to all of these terms, even if we use patterns.

**Comment by paolaroncaglia** [comment]
Closing as a duplicate of https://github.com/obophenotype/cell-ontology/issues/1262.
